mantidproject/mantid
# Re #20875: Modified HintingLineEdit to use normal text colour.

Modifies `HintingLineEdit` to use a modified colour palette which uses the normal text colour so that the text is readable on certain Linux distributions where the inactive text colour doesn't have enough contrast with the background.

**To test:**

_On a Linux Machine_
1. Open the reflectometry GUI `Interfaces->Reflectometry->ISIS Reflectometry`
2. Start typing in the options cell, as you type you should see a list of options rendered in a colour which is easily readable.

_And **also** on a Windows machine_
1. Open the reflectometry GUI `Interfaces->Reflectometry->ISIS Reflectometry`
2. Start typing in the options cell, as you type you should see (as before) a list of options is still rendered in a colour which is easily readable.



Fixes #20875. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
